generators/app/index.js

```diff
@@ -25,12 +25,14 @@ module.exports = Generator.extend({
     this.destinationRoot(this.options.projectName);
 
     // General
-    this.fs.copyTpl(
-      this.templatePath('.gitignore'),
-      this.destinationPath('.gitignore'), {
-        platform: this.options.platform
-      }
-    );
+    if (!this.options.gitInit) {
```

Comment:
Shouldn't this actually be `if (this.options.gitInit`? It's not generating a .gitignore for me locally.